css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
+        background-color: lightblue;
+        text-align: center;
+        
+      }
+
+      h1 {
+        color: blue;
+        align-items: center;
+      }
+
```

Comment:
- [ ] Please use font family and font size properties should be added so that the text size and length can be controlled.